### PR TITLE
Default the import base path to something sensible

### PR DESCRIPTION
**Pull request by Frassle**
This fixes go package generation so users don't _have_ to set the import base path explicitly in the go info part of the schema. This only really helps our packages (because we assume the path is "github.com/pulumi") but it fixes some of the generated test data and should also work for local SDKs, where they aren't _really_ at "github.com/pulumi" but also that it doesn't matter what URL is used as long as it's a valid one.

Fixes https://github.com/pulumi/pulumi/issues/13756

### CODE REVIEW

File: tests/testdata/codegen/output-funcs-edgeorder/go/myedgeorder/listConfigurations.go [hunk 1]
```diff
@@ -7,8 +7,8 @@ import (
 	"context"
 	"reflect"
 
+	"github.com/pulumi/pulumi-myedgeorder/sdk/go/myedgeorder/internal"
 	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
-	"internal"
```
Comment (tgummerer) [inline tests/testdata/codegen/output-funcs-edgeorder/go/myedgeorder/listConfigurations.go:11]: I was wondering if this change could break someone, but just importing `internal` is broken already, so I think the answer is no.

**Review by tgummerer — APPROVED** [review]

File: tests/testdata/codegen/output-funcs-edgeorder/go/myedgeorder/listConfigurations.go (hunk 1 above)
Comment (Frassle) [reply tests/testdata/codegen/output-funcs-edgeorder/go/myedgeorder/listConfigurations.go:11]: Aye, I think every published SDK would have been setting importBasePath anyway. This is mostly just helpful to trim our schemas for tests and to make local SDKs work.

**Comment by justinvp** [comment]
Looks related to https://github.com/pulumi/pulumi/issues/13756

**Comment by Frassle** [comment]
> Looks related to https://github.com/pulumi/pulumi/issues/13756

Ah yup, I thought we had an issue for this. This is a bit more plain than the suggestion in that issue, but I don't think we can use build info in general anyway as we move to plugin based sdk generation.

**Pull request merged**